catroid/src/catroid/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -32,7 +32,7 @@
 public final class FlavoredConstants {
 
 	public static final File DEFAULT_ROOT_DIRECTORY = new File(
-			Environment.getExternalStorageDirectory().getAbsolutePath() + "/Pocket Code");
+			CatroidApplication.getAppContext().getFilesDir().getParentFile().getAbsolutePath() + "/Programs");
```

Comment:
please dont do getParentFile()
it doesnt matter, its just deviating from what android actually wants you to do.